tests/test_fair_estimations.py

```diff
@@ -122,62 +122,51 @@ def linear_extrapolation(x, x_1, x_2, y_1, y_2):
     )
 
 
-def test_c_equivalence():
+@pytest.mark.parametrize(
+    "eta, expected",
+    [
+        pytest.param(0, xr.DataArray(np.array(5.0)), id="eta=0"),
```

Comment:
Are the numbers in the `xr.DataArray`s here calculated offline as answers to `c_equivalence` with those eta values?

Reply:
Yes. I calculated them on my machine using the code in the original test. Happy to change if you'd like, @kemccusker.

Reply:
all good, thank you!